examples/jupyter/pincell_depletion.ipynb

```diff
@@ -0,0 +1,1028 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "pyplot.plot(time, xe135, label=\"Xe135\")\n",
+    "pyplot.xlabel(\"Time [d]\")\n",
+    "pyplot.ylabel(\"Number of atoms - Xe135\");"
+   ]
+  },
+  {
+
```

Comment:
It looks like we don't actually document the units on `Operator.heavy_metal` in the docstring. Can you add `[g]` to the description of the `heavy_metal` attribute in the `Operator` docstring?